### PR TITLE
advice to nyima_mipham

### PR DESCRIPTION
**Pull request by JJM317**
Spelling error in the branch name. Couldn't figure out how to rename a branch. I guess this is ok since it gets deleted anyway later on?

### CODE REVIEW

File: _content/essays/advice-to-nyima_mipham.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+course: emptiness
+tags:
+  - meditation
+  - problems
+year: 2016
+publisher: "Lotsawa House"
+pages: 3
+---
+
+> Leave behind the analogies of foolish minds and modes of speech,
```
Comment (buddhist-uni) [inline _content/essays/advice-to-nyima_mipham.md:19]: Two spaces at the end of lines of poetry

File: _content/essays/advice-to-nyima_mipham.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+authors:
+  - mipham
+subcat: poetry
+translator: "Adam Pearcey"
+external_url: "https://www.lotsawahouse.org/tibetan-masters/mipham/advice-to-jigme-tenpe-nyima"
+drive_links:
+  - "https://drive.google.com/file/d/1t2KdPQHDMCfquidNXNwDAfKjly8QzSCC/view?usp=drive_link"
+course: emptiness
+tags:
+  - meditation
```
Comment (buddhist-uni) [inline _content/essays/advice-to-nyima_mipham.md:12]: no need for meditation, but should have `monastic-advice` for sure

**Review by buddhist-uni — COMMENTED** [review]
Yeah, branch names don't really matter so no worries 😁

**Comment by JJM317** [comment]
Just tried making the changes via VS. Let me know if they went through
correctly, please🤞.
Oh, and I made an additional change - The Mipham file was from the
epistemology folder, not the emptiness folder, so I fixed that. I saw your
note about the subcat:course. Will keep that in mind.
Kindly,
~Joe
--------------------

**Pull request merged**